src/pages/Lending/hooks/useLendingPositionData.tsx

```diff
@@ -44,6 +44,9 @@ export const useLendingPositionData = ({ accountId, assetId }: UseLendingPositio
       }
     },
     enabled: Boolean(accountId && assetId && poolAssetMarketData.price !== '0'),
+    refetchOnMount: true,
+    refetchInterval: 60_000,
+    refetchIntervalInBackground: true,
```

Comment:
informative question: do the refetch params honor `staleTime` or is `staleTime` only relevant to manual query function calls?

Reply:
As far as I understand it, refetch is just a manual refetch, similar to calling the query at component-level with the hook i.e would still honor `staleTime`.